lib/utils/constants.dart

```diff
@@ -1 +1,7 @@
+import 'package:flutter/material.dart';
+
 const String SochemIcon = "assets/sochem.jpeg";
+const kPrimaryColor = Color(0xFF3449c0);
+const kBackgroundColor = Color(0xFFd6e0f9);
+const cardImageColor = Color(0xFF6282d3);
+const double kDefaultPadding = 20.0;
```

Comment:
You can introduce spaces between image paths (strings), colors and numeric constants (eg pagging).

Reply:
Done